training_configs/vilbert_gqa_from_huggingface.jsonnet

```diff
@@ -0,0 +1,92 @@
+local model_name = "bert-base-uncased";
+local effective_batch_size = 128;
+local gpu_batch_size = 32;
+local num_gpus = 1;
+
+local construct_vocab = false;
+
+local vocabulary = if construct_vocab then {
+      // read the files to construct the vocab
+      "min_count": {"answers": 1}
+    } else {
+      // read the constructed vocab
+      "type": "from_files",
+      // CHANGE LOC
+      "directory": "/Users/jacksons/Projects/gqa-train/vocabulary/vilbert_gqa_train.vocab.tar.gz"
```

Comment:
Can you upload this to GCloud? If you don't know how or where, ping me.

Reply:
Should I keep the min_count size of 9 for this vocab? generating on beaker now

Reply:
Yes, let's stick with that number.

Generating vocab takes only a few minutes on my machine.